splunk-otel-android/src/test/java/com/splunk/rum/ThrottlingExporterTest.java

```diff
@@ -200,8 +209,12 @@ public void shouldDelegateShutdownCall() {
     }
 
     private static SpanData span(String component, Instant endTime) {
+        return span(component, endTime, "");
+    }
+
+    private static SpanData span(String component, Instant endTime, String additionalName) {
         return TestSpanData.builder()
-                .setName("test")
+                .setName("test" + additionalName)
```

Comment:
Is that `additionalName` actually used anywhere?

Reply:
Oh shoot. I added that while debugging a test and forgot to remove it. I'll remove it.